usr/share/rear/prep/NBKDC/default/400_prep_nbkdc.sh

```diff
@@ -1,38 +1,56 @@
 #
-# prepare stuff for NovaBACKUP DataCenter
+# prepare stuff for NovaStor DataCenter
 #
 
-# detect where and if DataCenter client is installed on the system running
-# REAR and include the needed bits in the recovery ISO
+# Note: unlike in C/C++, in bash `if function param..` uses the then branch, if
+# the function returns 0
+function is_nbkdc_dir() {
+  local candidate="$1"
+  if [ -x $candidate/rcmd-executor/rcmd-executor ]; then
```

Comment:
This is overcomplicated because when there is no explicit 'return'
a function returns the exit code of its last command so plain
```
function is_nbkdc_dir() {
  local candidate="$1"
  test-x $candidate/rcmd-executor/rcmd-executor
}
```
should be enough.

Reply:
Thank you for your suggestion. I have pushed a commit to simplify them.